helpers/utils.go

```diff
@@ -0,0 +1,77 @@
+package helpers
+
+import (
+	"fmt"
+	"github.com/go-yaml/yaml"
+)
+
+// parse given string in YAML format
+func ParseYamlString(value string) (interface{}, error) {
+	var result interface{}
+
+	err := yaml.Unmarshal([]byte(value), &result)
+	// returns value when unmarshal fails
+	if err != nil {
+		return value, err
+	}
+
+	if result == "" {
+		return value, nil
+	}
+
+	return result, nil
+}
+
+// convert value to a string.
+func InterfaceToString(value interface{}) string {
+	if value == nil {
+		return ""
+	}
+	return fmt.Sprintf("%v", value)
+}
+
+// converts value to a string or YAML string
```

Comment:
// if the value is already a string return it, otherwise convert to the YAML representation